backend/src/api/scans.ts

```diff
@@ -49,85 +49,36 @@ interface ScanSchema {
 }
 
 export const SCAN_SCHEMA: ScanSchema = {
-  testProxy: {
-    type: 'fargate',
-    isPassive: false,
-    global: true,
-    description: 'Not a real scan, used to test proxy'
-  },
-  test: {
-    type: 'fargate',
-    isPassive: false,
-    global: true,
-    description: 'Not a real scan, used to test'
-  },
-  censys: {
-    type: 'fargate',
-    isPassive: true,
-    global: false,
-    description: 'Passive discovery of subdomains from public certificates'
-  },
   amass: {
     type: 'fargate',
     isPassive: false,
     global: false,
     description:
       'Open source tool that integrates passive APIs and active subdomain enumeration in order to discover target subdomains'
   },
-  findomain: {
-    type: 'fargate',
-    isPassive: true,
-    global: false,
-    description:
-      'Open source tool that integrates passive APIs in order to discover target subdomains'
-  },
-  portscanner: {
-    type: 'fargate',
-    isPassive: false,
-    global: false,
-    description: 'Active port scan of common ports'
-  },
-  wappalyzer: {
-    type: 'fargate',
-    isPassive: true,
-    global: false,
-    cpu: '1024',
-    memory: '4096',
-    description:
-      'Open source tool that fingerprints web technologies based on HTTP responses'
-  },
-  shodan: {
-    type: 'fargate',
-    isPassive: true,
-    global: false,
-    description:
-      'Fetch passive port, banner, and vulnerability data from shodan',
-    cpu: '1024',
-    memory: '8192'
-  },
-  sslyze: {
+  censys: {
     type: 'fargate',
     isPassive: true,
     global: false,
-    description: 'SSL certificate inspection'
+    description: 'Passive discovery of subdomains from public certificates'
   },
-  censysIpv4: {
+  censysCertificates: {
```

Comment:
@Matthew-Grayson Instead of changing the order here, can we instead change the frontend code so it alphabetizes the keys of this dictionary? Not sure if we want to rely on dictionary key order.

Reply:
Done.